src/driver/sqlserver/SqlServerDriver.ts

```diff
@@ -891,7 +891,11 @@ export class SqlServerDriver implements Driver {
     /**
      * Returns true if driver supports RETURNING / OUTPUT statement.
      */
-    isReturningSqlSupported(): boolean {
+    isReturningSqlSupported(returningType: ReturningType): boolean {
+        if (returningType === 'insert') {
+          return false;
```

Comment:
SqlServer supports "insert returning" ("insert output") and TypeORM already implements it. So this change can be undone.